README.md

```diff
@@ -48,7 +48,8 @@ email-verification only, is to be found
 
 ### Running server
 
-     $ docker-compose run --rm --service-ports serve
+     $ docker-compose up serve celery
+     $ (Without celery, the old pdb-friendly way: docker-compose run --rm --service-ports serve)
```

Comment:
@szabozoltan69 Maybe we can use this as well.

https://github.com/the-deep/deeper/blob/35f01f977293d1c26d0667bd1ea6458dcf4e4397/docker-compose.yml#L7-L10